app/cosi/README.md

```diff
@@ -0,0 +1,338 @@
+[![Build Status](https://travis-ci.org/dedis/cothority.svg?branch=master)](https://travis-ci.org/dedis/cothority)
```

Comment:
I'm OK with having the instructions in the Wiki and a link pointing to the wiki in the app/cosi/README.md. We could do that for all apps, so we have a central place where all instructions / docs can be read.